src/AdminForm.php

```diff
@@ -1106,6 +1106,40 @@ private function buildContributionTab() {
       '#required' => TRUE,
     ];
 
+    // Billing Address
+    $n = wf_crm_aval($this->data, "billing:number_number_of_billing", 0);
+    $this->form['contribution']['sets']["billing_1_number_of_billing"] = [
+      '#type' => 'select',
+      '#title' => t('Enable Billing Address?'),
+      '#default_value' => $n,
+      '#options' => [t('No'), t('Yes')],
+      '#prefix' => '<div class="number-of">',
+      '#suffix' => '</div>',
+      '#description' => t('Enable this section if you want billing address to be sent to the payment processor.'),
+    ];
```

Comment:
Yes to require all billing fields as Civi is expecting them (even if the payment processor couldn't care less). Except for Billing Middle Name - that's not required.